src/components/Datepicker/Datepicker.tsx

```diff
@@ -66,7 +53,7 @@ interface BaseDatepickerProps extends UseDatepickerProps {
     locale: Locale;
 }
 
-export const BaseDatepicker: FC<BaseDatepickerProps> = ({
+const BaseDatepicker: FC<BaseDatepickerProps> = ({
```

Comment:
please remove the `BaseDatepcikerProps` from the line 61, to avoid double prop typing